docs/feature-store/sources-targets.md

```diff
@@ -200,6 +204,14 @@ It supports low latency data retrieval based on key access, making it ideal for
 
 The combination of a NoSQL target with the storey engine does not support features of type string with a value containing both quote (') and double-quote (").
 
+## Redis target
```

Comment:
both "Redis target" and "RedisNoSql target"?
Also I suggest we consistently:
1. use the class name as the section name - what do you think? So for example - RedisNoSqlTarget
2. First explain the class parameters, then describe the associated datastore profile